tests/test_base.py

```diff
@@ -70,3 +70,15 @@ def test_finalize_interface(self):
         """
         visualizer = Visualizer()
         self.assertIs(visualizer.finalize(), visualizer.ax)
+
+    def test_size_property(self):
+        """
+        Test the size property on the base Visualizer
+        """
+        visualizer = Visualizer()
+        self.assertIsNone(visualizer._size)
+        self.assertIsNotNone(visualizer.size)
+
+        visualizer.size = 42
```

Comment:
The size property should actually be a tuple - I get what you're doing in this test, but it also serves as documentation, I recommend using `(1080, 720)` -- a pretty standard resolution.